ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -265,9 +275,9 @@ function WorkloadCvesOverviewPage() {
         // Reset all filters, sorting, and pagination and apply to the current history entry
         pagination.setPage(1);
         setSearchFilter({});
-        if (activeEntityTabKey === 'CVE') {
+        if (mode === 'WITHOUT_CVES') {
             setActiveEntityTabKey('Image');
-            sort.setSortOption(getDefaultSortOption('Image'));
+            sort.setSortOption(getDefaultZeroCveSortOption('Image'));
```

Comment:
Not related to this change exactly, but fixes a semi-related bug where the sort fields are set incorrectly.